docs/how-to-guides/develop-a-tool/add-a-tool-icon.md

```diff
@@ -0,0 +1,64 @@
+# Add a Tool Icon
+A tool icon serves as a graphical representation of your tool in the user interface (UI). Follow this guidance to add a custom tool icon when developing your own tool package.  
+
+Adding a custom tool icon is optional. If you do not provide one, the system uses a default icon.
+
+## Prerequisites
+
+- Create a tool project as described in [Create and Use Tool Package](create-and-use-tool-package.md).
+
+- Install the dependent library:
+
+  ```
+  pip install pillow
+  ```
+
+- Prepare a custom tool icon image in `PNG` or `JPG` format.
```

Comment:
- Prepare a custom tool icon image. The image should meet following requirements:
  - <format>.
  - 16x16 pixel image to avoid distortion from resizing.
  - Avoid complex images with much detail or contrast as they may not resize well.
You could see <path-to-example-icon> for example. 
![]()

> [!Note] The script in the next section verifies the image size and resizes it to 16x16 pixels if needed.

Reply:
Changed, thanks.